src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -89,6 +92,7 @@ internal async void SetLoadingDone()
             loadingTimer.Stop();
             loadingTimer = null;
             await webView.CoreWebView2.ExecuteScriptAsync($"window.setLoadingDone()");
+            await webView.CoreWebView2.ExecuteScriptAsync($"window.setTotalLoadingTime('Total loading time: {totalLoadingTime}ms')");
```

Comment:
Do we need to localize this?

Reply:
Yes, I will create a resource for it. Thanks!